lib/client.js

```diff
@@ -343,7 +359,7 @@ Client.prototype = {
       throw new Error('The get method accepts a string or array of strings.');
     }
 
-    return wrappedPostMessage.call(this, 'get', paths);
+    return wrappedPostMessage.call(this, 'get', paths).then(throwUnlessBulk.bind(null, path));
```

Comment:
well, in the current env. yes, but if we are going to change behaviour, i would change the returned result completely and make it more consistent.